smqtk_iqr/web/search_app/modules/iqr/iqr_search.py

```diff
@@ -80,7 +81,10 @@ def get_default_config(cls):
 
     # noinspection PyMethodOverriding
     @classmethod
-    def from_config(cls, config, parent_app):
+    def from_config(
+        cls: Type[T], config: Dict[str, Any],
+        parent_app: IqrSearchDispatcher
+    ) -> T:
```

Comment:
`smqtk_iqr/web/search_app/modules/iqr/iqr_search.py:84: error: Signature of "from_config" incompatible with supertype "Configurable"`

Unsure exactly what the problem is here

Reply:
The function signature is different than the method being overridden. It's a hack that flew under the radar due to the previous lack of type checking.

Reply:
The other time this has happened, in `smqtk-descriptors` `DescriptorElement` type this method was given a `# type: ignore`. We may have to go with such a sledge hammer here given that we don't have the bandwidth of a refactor here.